include/actions/AddTallyAction.h

```diff
@@ -40,11 +40,7 @@ class AddTallyAction : public MooseObjectAction
    */
   void addMeshTally(unsigned int instance = 0, const Point & translation = Point(0.0, 0.0, 0.0));
 
-  /**
-   * This function adds a a tally which may be an offset into an array
-   * of translations with index instance. Used for translated mesh tallies
-   * that all use the same base mesh for the spatial filter.
-   */
+  /// This function adds a non-mesh tally.
```

Comment:
Can this doc string be made more specific? Is this true? If so, it may be clearer for folks reading Doxygen

`This function adds a cell tally`

Reply:
At the moment it's true in that the only non-translatable tallies in Cardinal are `CellTally` objects. My thought is that if other tally objects (such as ones which implement a ZernikeFilter) would also get added by that function.

I can definitely try to make the doc string clearer though. Adding a list of tallies which would be added by that function and a better description of what it does would help a lot compared to a single line.